src/qt/transactiondesc.cpp

```diff
@@ -140,13 +140,6 @@ QString TransactionDesc::toHTML(CWallet *wallet, CWalletTx &wtx, TransactionReco
             strHTML += "(" + tr("not accepted") + ")";
         strHTML += "<br>";
     }
-    else if (nNet > 0)
-    {
-        //
-        // Credit
-        //
-        strHTML += "<b>" + tr("Credit") + ":</b> " + BitcoinUnits::formatHtmlWithUnit(unit, nNet) + "<br>";
-    }
```

Comment:
With this block gone, line 227 gets applied instead, which has the same output.
Another way to do this would be to create an  if (fAllToMe)  block, which may be clearer in the code.